sorbet/rbi/gems/actionpack.rbi

```diff
@@ -1765,6 +1765,10 @@ class ActionDispatch::RemoteIp::GetIp
   def ips_from(header); end
   def to_s; end
 end
+module ActionController::ApiRendering
```

Comment:
Doing this just causes a bunch of churn (in `sorbet/rbi/gems`).  The RBI files do not generate in a cannonical way, so we get changes in the repo with no meaningful impact.